pkgs/by-name/bu/budgie-session/package.nix

```diff
@@ -38,12 +40,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-mz+Yh3NK2Tag+MWVofFFXYYXspxhmYBD6YCiuATpZSI=";
   };
 
+  outputs = [
+    "out"
+    "man"
+  ];
+
   patches = [
     (substituteAll {
       src = ./fix-paths.patch;
-      gsettings = "${glib.bin}/bin/gsettings";
-      dbusLaunch = "${dbus.lib}/bin/dbus-launch";
-      bash = "${bash}/bin/bash";
+      gsettings = lib.getExe' glib "gsettings";
+      dbusLaunch = lib.getExe' dbus "dbus-launch";
```

Comment:
(Not a request change) For future reference

https://github.com/NixOS/nixpkgs/blob/452a2137e0e33173e87e2bba01af8394ca8f68b1/pkgs/development/libraries/dbus/default.nix#L108-L109